react/tic-tac-toe/src/redux/tictactoe/actions.js

```diff
@@ -1,11 +1,31 @@
+import matchesService from '../../services/matchesService';
+import { parseMatchesResponse } from '../../utils/mappers';
+
 export const actions = {
-  PLACE_PIECE: '@@TICTACTOE/PLACE_PIECE'
+  GET_HISTORIC_SUCCESS: '@@TICTACTOE/GET_HISTORIC_SUCCESS',
+  GET_HISTORIC_LOADING: '@@TICTACTOE/GET_HISTORIC_LOADING',
+  GET_HISTORIC_ERROR: '@@TICTACTOE/GET_HISTORIC_ERROR'
 };
 
 const actionsCreators = {
-  placePiece: () => ({
-    type: actions.PLACE_PIECE
-  })
+  getHistoric: data => ({
+    type: actions.GET_HISTORIC_SUCCESS,
+    payload: data
+  }),
+  getHistoricError: error => ({
+    type: actions.GET_HISTORIC_ERROR,
+    payload: error
+  }),
+  getHistoricLoading: () => dispatch => {
```

Comment:
The loading state is missing, you should start by setting that. Probably the spinner is never should because of that. Also, I suggest moving the actionCreators that aren't supposed to be accessed outside this file into a "private" constant:

```
const privateActionCreators = {
  getHistoric: () => ({
    type: actions.GET_HISTORIC_LOADING,
    payload: data
  }),
  getHistoricError: error => ({
    type: actions.GET_HISTORIC_ERROR,
    payload: error
  }),
  getHistoricSuccess: data => ({
    type: actions.GET_HISTORIC_SUCCESS,
    payload: data
  })
}

export const actionCreators = {
  getHistoric: () => dispatch => {
    dispatch(privateActionCreators.getHistoricLoading());
    const response = matchesService.getHistoric();
    if (response.ok) {
      const { data } = response;
      const parsedResponse = data.map(parseMatchesResponse);
      dispatch(privateActionCreators.getHistoric(parsedResponse));
    } else {
      dispatch(privateActionCreators.getHistoricError(response.error));
    }
  }
}
```